controllers/widget.js

```diff
@@ -104,6 +104,19 @@ function applyProperties(properties) {
     _applyInnerProperties(_properties);
 }
 
+function setAccessibilityLabel(label, hint) {
+
+    $.inner.setAccessibilityLabel(label);
+    $.outer.setAccessibilityLabel(label);
```

Comment:
No, in fact you only touch one of them, so that's the only way I found to be sure the label is read 
I might be wrong, but all my tests went well